scripts/obtain/dotnet-install.sh

```diff
@@ -534,17 +534,18 @@ get_absolute_path() {
 copy_files_or_dirs_from_list() {
     eval $invocation
 
-    local root_path=$(remove_trailing_slash $1)
-    local out_path=$(remove_trailing_slash $2)
-    local override=$3
-    local override_switch=$(if [ "$override" = false ]; then printf -- "-n"; fi)
+    local root_path="$(remove_trailing_slash "$1")"
+    local out_path="$(remove_trailing_slash "$2")"
+    local override="$3"
+    local override_switch="$(if [ "$override" = false ]; then printf -- "-n"; fi)"
     
     cat | uniq | while read -r file_path; do
-        local path=$(remove_beginning_slash ${file_path#$root_path})
-        local target=$out_path/$path
+        local path="$(remove_beginning_slash "${file_path#$root_path}")"
+        local target="$out_path"/"$path"
```

Comment:
Why quote each part? Doesn't this work in a way that's expected: local target="$out_path/$path"?

Reply:
It does--I think I did that because I was searching around for $word patterns and quote-surrounding them. Can definitely be changed.

Reply:
Yes, please.